components/input/input.html

```diff
@@ -0,0 +1,196 @@
+    <link rel="stylesheet" href="../../docs.css">
+    <link rel="stylesheet" href="../../landing.css">
+</head>
+<body>
+    <header class="navbar" >
+        <p class="navbar-brand">
+            <a href="../../index.html"  >Sinister UI</a>
+            <small class="subtitle2 small-text" >V1.0</small>
+        </p>        
+        <nav>
+            <a href="../../docs.html" class="btn btn-text teal" >Docs</a>        
+        </nav> 
+
+    </header>
+    <main class="docs-container">
+        <aside class="sidebar">
+            <ul>
+                <span class="headline4">Get Started</span>
+                <a href="#" class="btn btn-text btn-link teal button-txt" >
+                    <li>
+                        Installation
+                    </li>
+                </a>
+                <a href="#" class="btn btn-text btn-link teal button-txt" >
+                    <li>
+                        Color
+                    </li>
+                </a>
+                <a href="#" class="btn btn-text btn-link teal button-txt" >
+                    <li>
+                        Typography
+                    </li>
+                </a>
+            </ul>
+            <hr class="line-horz">
+            <ul>
+                <span class="headline4">Components</span>
+                <a href="../alert/alert.html" class="btn btn-text btn-link teal button-txt " >
+                    <li>
+                        Alert
+                    </li>
+                </a>
+                <a href="../avatar/avatar.html" class="btn btn-text btn-link teal button-txt" >
+                    <li>
+                        Avatar
+                    </li>
+                </a>
+                <a href="../badge/badge.html" class="btn btn-text btn-link teal button-txt" >
+                    <li>
+                        Badge
+                    </li>
+                </a>
+                <a href="../button/button.html" class="btn btn-text btn-link teal button-txt " >
+                    <li>
+                        Button
+                    </li>
+                </a>
+                <a href="../card/card.html" class="btn btn-text btn-link teal button-txt " >
+                    <li>
+                        Card
+                    </li>
+                </a>
+                <a href="../image/image.html" class="btn btn-text btn-link teal button-txt" >
+                    <li>
+                        Image
+                    </li>
+                </a>
+                <a href="../input/input.html" class="btn btn-text btn-link teal button-txt link-active" >
+                    <li>
+                        Input
+                    </li>
+                </a>
+            </ul>
+        </aside>
+        
+        <div class="docs-body">
+            <h1 class="headline1 text-center">Input</h1>
+            <p class="body1">Input fields allow users to enter text into a UI. They typically appear in forms and dialogs.</p>
```

Comment:
What is body1 and heading1?

Reply:
I am keeping typo in the form of
healdine1.....headline6, body1,body2, subtitle1,subtitle2
just like materialUI